sdk/ai/ai-inference-rest/test/public/browser/streamingChat.spec.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { createRecorder, createModelClient } from "../utils/recordedClient.js";
+import { Recorder } from "@azure-tools/test-recorder";
+//import { createSseStream } from "@azure/core-sse";
+import { assert, beforeEach, afterEach, it, describe } from "vitest";
+import { ModelClient } from "../../../src/index.js";
+
+describe("chat test suite", () => {
+  let recorder: Recorder;
+  let client: ModelClient;
+
+  beforeEach(async (context) => {
+    recorder = await createRecorder(context);
+    client = await createModelClient(recorder);
+  });
+
+  afterEach(async () => {
+    await recorder.stop();
+  });
+  it("chat streaming test", async function () {
+    const response = await client.path("/chat/completions").post({
+      body: {
+        messages: [
+          { role: "user", content: "How many feet are in a mile?" },
+        ],
+        stream: true
+      }
+    }).asBrowserStream();
+
+    assert.equal(response.status, "200");
+    const stream = response.body;
```

Comment:
Did you consider passing this stream to `createSseStream`?

Reply:
I will test that out.

Reply:
[Updated](https://github.com/Azure/azure-sdk-for-js/pull/29863/commits/f7a7f389f671336f4cf69e0119655c153cb4ad11)